libdnf.spec

```diff
@@ -223,7 +253,7 @@ popd
 
 %find_lang %{name}
 
-%if 0%{?rhel} && 0%{?rhel} <= 7
+%if (0%{?rhel} && 0%{?rhel} <= 7) || 0%{?mageia} || 0%{?is_opensuse}
```

Comment:
Mageia uses file triggers, so this is not required.

Reply:
> Also, my authorship of the patch isn’t preserved in this PR…
I'm aware of it, I'm going to fix it.

Reply:
> Mageia uses file triggers, so this is not required.

In that case we need to skip %ldconfig_scriptlets there, because (IIRC) the build was failing on this macro.

Reply:
Let me see if I can make that macro exist on Mageia, then…

Reply:
I've added them to Cauldron, they should show up in the next ~48 hours as mirrors sync: http://svnweb.mageia.org/packages?view=revision&revision=1422833

Reply:
0%{?mageia} is now removed